internal/csi-addons/rbd/network_fence_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbd
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	fence "github.com/csi-addons/spec/lib/go/fence"
+)
+
+// TestFenceClusterNetwork is a minimal test for the
+// FenceClusterNetwork() procedure. During unit-testing, there is no Ceph
+// cluster available, so actual operations can not be performed.
+func TestFenceClusterNetwork(t *testing.T) {
+	t.Parallel()
+
+	controller := NewFenceControllerServer()
+
+	req := &fence.FenceClusterNetworkRequest{
+		Parameters: map[string]string{},
+		Secrets:    nil,
+		Cidrs:      nil,
+	}
+
+	_, err := controller.FenceClusterNetwork(context.TODO(), req)
+	assert.Error(t, err)
```

Comment:
assert for the specific error messages what you are expecting.

Reply:
I think any error other than `nil` should be considered a test fail?

Reply:
unit tests should test for the specific error message in most of the cases to make sure things are working as expected.

Reply:
You might be able to check for `status.InvalidArgument`, but not much more? Not sure if there is much value in that.